README.md

```diff
@@ -35,6 +35,8 @@ A searchable sort-able and group-able version of the list of companies below is
 - [HiPages](https://hipagesgroup.com.au/careers/) | Sydney CBD, Sydney | PHP, NodeJs, MySQL, Swift, Objective-C
 - [HireUp](https://hireup.com.au/careers/) | Greenwich, Sydney | NodeJs, Javascript, React, Mongo DB
 - [IRESS](https://www.iress.com/au/company/join-us/) | Melbourne CBD, Melbourne; Sydney CBD, Sydney | Reactjs, Python, JavaScript, TypeScript, AWS
+- [Isentia](https://www.isentia.com/careers/) | Sydney CBD, Sydney; Melbourne CBD | Nodejs, Angularjs, Python, AWS, Docker 
+
```

Comment:
Looks like there is 1 extra unneeded blank line, would you be able to remove it @ghaleroshan ?